simulator/models/ropod/ropod.sdf

```diff
@@ -226,7 +226,24 @@
      
       
     </sensor>
-      
+
+
+    <!-- ********************** bumper ***************************** -->
+    <sensor type="contact" name="gazebo_base_bumper">
+        <update_rate>1000.0</update_rate>
```

Comment:
in the real robot the update rate is 100Hz

Reply:
Ah, didn't know that, thanks. I've just decreased it to 100Hz.